examples/c/mass_spring_qpdunes.c

```diff
@@ -29,11 +29,12 @@
 #include "acados/ocp_qp/ocp_qp_common_frontend.h"
 #include "acados/ocp_qp/ocp_qp_qpdunes.h"
 #include "acados/utils/create.h"
+#include "acados/utils/print.h"
 #include "acados/utils/timing.h"
 #include "acados/utils/types.h"
 #include "acados/utils/mem.h"
 
-#define NREP 100
+#define NREP 1
```

Comment:
this one also

Reply:
I left this so that the qpDUNES problem remains apparent. See the comment below.